TypeRaceAir/race.py

```diff
@@ -0,0 +1,63 @@
+import typing
+
+
+class Race():
+
+    def __init__(self, sentece: str, players: typing.List[Player]):
+        self.sentece = sentece
+        self.players = players
+
+
+class Player():
+    pass
+
+
+class PlayerRaceResult():
```

Comment:
Please document each object in general. Why does it exist? Where it will be used? What does it do?